tests/fgpyo/test_sequence.py

```diff
@@ -54,3 +54,37 @@ def test_homopolymer(bases: str, expected_hp_len: int) -> None:
 )
 def test_gc_content(bases: str, expected_gc_content: float) -> None:
     assert gc_content(bases) == expected_gc_content
+
+
+@pytest.mark.parametrize(
+    "bases, n, expected_length",
+    [
+        ("", 2, 0),
+        ("", 1, 0),
+        ("A", 1, 1),
+        ("A", 2, 0),
+        ("ACGT", 2, 2),
+        ("AACCGGTT",2,  2),
+        ("TTTTCCCGGA",2,  4),
+        ("ACCGGGTTTT",2,  4),
+        ("GTGTGTAAAA", 2, 6),
+        ("GTGTGTGAAAA",2,  6),
+        ("GTGTGTACACACAC",2,  8),
+        ("GTGTGTGACACACAC",2,  8),
```

Comment:
```suggestion
        ("AACCGGTT", 2, 2),
        ("TTTTCCCGGA", 2, 4),
        ("ACCGGGTTTT", 2, 4),
        ("GTGTGTAAAA", 2, 6),
        ("GTGTGTGAAAA", 2, 6),
        ("GTGTGTACACACAC", 2, 8),
        ("GTGTGTGACACACAC", 2, 8),
```

Have you run `ruff`?

Reply:
Our actions (and `check.sh`, presumably) don't run `ruff` on the `tests/` directory. I'll open a quick PR to fix

Reply:
#163